### PR TITLE
Details pop up window- a popup window for projects

### PR DESCRIPTION
**Pull request by Fsher07**
Created a popup window that includes details of project. Reachable when click on "see project" buttons.. Stored object in an array. Created elements dynamically via JavaScript function.

### CODE REVIEW

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @Fsher07 👋,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Status: Invalid Code Review Request



_**You have submitted a project with incomplete linter files. Kindly check [this](https://github.com/microverseinc/linters-config/tree/master/html-css-js) and add appropriate files.**_

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -11,6 +11,7 @@
 </head>
```
[the remark this answers is not in the document]
Comment (Fsher07) [reply index.html:11]: Thank you for your feedback. Now,I added Eslint file and fix errors.

File: index.html [hunk 2]
```diff
@@ -139,7 +143,7 @@ <h3 class="project-titles ProjectPage-h2 none-btn">Data Dashboard<br> Healthcare
           <a class="Tags" href="#">Google</a>
         </li>
       </ul>
-      <button type="button" class="projectbtn btn">See Project</button>
```
Comment (mike2611) [inline index.html:142]: - [ ] Great work in the mobile version of your project your pop-up looks great, however, in your desktop version, your pop-up is out center. please fix this and make sure to follow the template in Figma 
image

**Review by mike2611 — CHANGES_REQUESTED** [review]
Hi @Fsher07 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -139,7 +143,7 @@ <h3 class="project-titles ProjectPage-h2 none-btn">Data Dashboard<br> Healthcare
...
+      <button type="button" class="projectbtn btn project-open-button">See Project</button>
     </div>  
     <div class="card card2 ProjectPage">
       <h3 class="project-titles ProjectPage-h2">Data Dashboard<br> Healthcare</h3>
```
[the remark this answers is not in the document]
Comment (Fsher07) [reply index.html:149]: Excuse me, I didn't get the point. I created an array that stores objects of the pop-up windows and I used those objects to create a project window.  Or, should I remove the project sections elements from index.html and create them with javascript ?

[the remark this answers is not in the document]
Comment (mike2611) [reply index.html:149]: Hi @Fsher07, yes you should remove the hardcoded cards from your project section, and create your cards using the data of your object dynamically.

[the remark this answers is not in the document]
Comment (mike2611) [reply index.html:149]: You can do this by looping through your array of objects and creating a new element for each one of your objects inside your array, and then appending each element to your projects section.

[the remark this answers is not in the document]
Comment (mike2611) [reply index.html:149]: So for example, if you have only six objects in your array your projects section only needs to show six elements as this are created dynamically. 

image

**Review by elyor-doniyorov — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
- [OPTIONAL] To make it more informative, kindly provide a project `description`, `screenshot`, `how to set up the project` section in your Readme.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**